README.md

```diff
@@ -8,6 +8,21 @@ This is the client library backing the [Dataflux Dataset for Pytorch](https://gi
 
 The fast list component of this client leverages Python multiprocessing to parallelize the listing of files within a GCS bucket. It does this by implementing a workstealing algorithm, where each worker in the list operation is able to steal work from its siblings once it has finished all currently slated listing work. This parallelization leads to a real world speed increase up to 10 times faster than sequential listing. Note that paralellization is limited by the machine on which the client runs, and optimal performance is typically found with a worker count that is 1:1 with the available cores. Benchmarking has demonstrated that the larger the object count, the better Dataflux performs when compared to a linear listing.
 
+### Example Code
+```python
+from dataflux_core import fast_list
+
+number_of_workers = 20
+project = "MyProject"
+bucket = "TargetBucket"
+target_folder_prefix = "folder1/"
+
+print("Fast list operation starting...")
+list_result = fast_list.ListingController(
+    number_of_workers, project, bucket, prefix=target_folder_prefix
```

Comment:
here and below: it will be best to reduce confusion by explicitly naming the function arguments.

[go/python-tips/064](https://goto.google.com/python-tips/064)